mhs/spineroutelookup/lookup/tests/test_sds_client.py

```diff
@@ -58,18 +59,20 @@ class TestSDSClient(TestCase):
         client = mocks.mocked_sds_client()
 
         result = await client._accredited_system_lookup(ODS_CODE, INTERACTION_ID)
+
         self.assertIsNotNone(result)
         self.assertIsNotNone(result[0]['attributes']['nhsMHSPartyKey'] == 'A91461-9199084')
```

Comment:
I think it's worth adding a check of the `uniqueIdentifier` value here:
```suggestion
         self.assertEqual(result[0]['attributes']['nhsMHSPartyKey'], 'A91461-9199084')
         self.assertEqual(result[0]['attributes']['uniqueIdentifier'], 'A91461-9199084')
```

Reply:
Had to update the mock server files whilst I was at it as well since I realised my asid was in there